opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -104,7 +104,7 @@ fn calculate_hash(values: &[KeyValue]) -> u64 {
 
 impl AttributeSet {
     fn new(mut values: Vec<KeyValue>) -> Self {
-        values.sort_unstable();
+        values.sort_unstable_by(|a, b| a.key.cmp(&b.key));
```

Comment:
will this be a breaking change if the user is sorting the KeyValue using the sort/sort_unsable before calling the metrics API?

Reply:
Yes, but I don't expect anyone to be doing that. We don't ask to sort the metric attributes before recording a measurement.

Reply:
Yes agree. Good to have changlog entry just in case someone is doing that :)

Reply:
Updated.